src/messenger-main.tsx

```diff
@@ -34,6 +36,7 @@ export class Container extends React.Component<Properties> {
   componentDidUpdate(prevProps: Properties): void {
     if (this.idChanged(prevProps)) {
       this.props.setActiveConversationId(this.conversationId);
+      this.props.validateActiveConversation();
```

Comment:
This should happen in a saga. There should be no business logic in the components.